internal/http/activity.go

```diff
@@ -1,33 +1,31 @@
-package routes
+package http
 
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"net/http"
 	"sublinks/sublinks-federation/internal/activitypub"
 	"sublinks/sublinks-federation/internal/lemmy"
-	"sublinks/sublinks-federation/internal/log"
-
-	"fmt"
 
 	"golang.org/x/text/cases"
 	"golang.org/x/text/language"
 
 	"github.com/gorilla/mux"
 )
 
-func SetupActivityRoutes(r *mux.Router) {
-	r.HandleFunc("/activities/{action}/{id}", getActivityHandler).Methods("GET")
+func (s Server) SetupActivityRoutes() {
+	s.HandleFunc("/activities/{action}/{id}", s.getActivityHandler).Methods("GET")
 }
 
-func getActivityHandler(w http.ResponseWriter, r *http.Request) {
+func (s Server) getActivityHandler(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	var content []byte
 	switch vars["action"] {
 	case "create":
-		obj, err := GetPostActivityObject(vars["id"])
+		obj, err := s.GetPostActivityObject(vars["id"])
 		if err != nil {
-			log.Error("Error reading object", err)
+			s.Error("Error reading object", err)
```

Comment:
Maybe it's just me, but this feels really weird (essentially calling `server.Error` doesn't translate to "log an error")